app/api/video/serializers.py

```diff
@@ -0,0 +1,16 @@
+from rest_framework import serializers
+
+from api.video.models import Video
+
+
+class VideoModelSerializer(serializers.ModelSerializer):
+    """
+    Model serializer for model video
+    """
+
+    file = serializers.FileField(required=False, write_only=True)
+
+    class Meta:
+        model = Video
+        fields = '__all__'
+        read_only_fields = ['bucket_path']
```

Comment:
what about delete_time and upload_date? they read only too

Reply:
yes, my bad

Reply:
done